emc2/simulator/main.py

```diff
@@ -159,6 +165,11 @@ def make_simulated_data(model, instrument, N_columns, do_classify=False, unstack
                         model, hyd_type, is_conv=True,
                         qc_flag=False, use_rad_logic=use_rad_logic, parallel=parallel, chunk=chunk)
 
+    # Skip moment calculations and return only subcolumn-distributed q and N
+    if subcol_gen_only:
```

Comment:
@mckenna-stanford, we can't finalize and unstack the dataset in that `subcol_gen_only` option, so I suppose the lease inelegant option would be an else statement instead of the `return model` statement that will end right before the finalization option. Other, more elegant ideas are welcome.